5.2curlcommands.md

```diff
@@ -579,6 +579,46 @@ curl -X POST -H 'Content-type: application/json' http://localhost:8080/api/admin
 }'
 ```
 
+### ELN Previewer Requirements:

```

Comment:
Since this is use of the ZipPreviewer for another mimetype, I'd suggest not repeating all of the instructions and having a separate header, etc. Perhaps changing "Zip Preview Requirements" above to just "Zip Previewer" and adding a line about it being useable with *.zip and *.eln files would be enough. If there's a link to info about the .eln format, that could be added too. 
If that's clear, please go ahead and make the changes. If not, I can merge this and wordsmith afterwards.

Reply:
I wouldn't mind if you did what you suggest after merge. Cheers!